builtin/providers/aws/resource_aws_launch_configuration.go

```diff
@@ -206,3 +461,9 @@ func resourceAwsLaunchConfigurationDelete(d *schema.ResourceData, meta interface
 
 	return nil
 }
+
+func readBlockDevicesFromLaunchConfiguration(d *schema.ResourceData, launchConfiguration *autoscaling.LaunchConfiguration, autoscalingconn *autoscaling.AutoScaling) map[string]interface{} {
+	var blockDevices = make(map[string]interface{})
+	// Need to figure out how to determine the various instance types.
```

Comment:
@jwaldrip I'm recommending pretty much copying the [schema from instances](https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_instance.go#L149-L301) and the [block device handling code](https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_instance.go#L722-L829). I think it should translate pretty decently to the LC APIs.